schema/project-level/project-schema.json

```diff
@@ -1303,6 +1304,27 @@
           "uniqueItems": true,
           "minItems": 1
         },
+        "details": {
```

Comment:
I find `details` to be a bit of a schema anti-pattern, so I would prefer not to mirror OCDS here.

If kept, its current `description` is specific to OCDS.

Reply:
How about we just have `classifications` directly on `Organization` instead? I don't know why it's nested under `details` in OCDS.

Reply:
Yes, let's put it direct. The original issue is https://github.com/open-contracting/standard/issues/369 but there's not much motivation for having a `details` object.

Reply:
Done in https://github.com/open-contracting/infrastructure/pull/445/commits/59f5b6b7c85d88e58bddb063fa8138e347817aac